esprima.js

```diff
@@ -737,6 +738,13 @@
                     extra.openCurlyToken = extra.tokens.length;
                 }
             }
+
+            if (code === 123) {
+                state.curlyStack.push('{');
+            } else if (code === 125) {
```

Comment:
I know the code is inconsistent elsewhere, but can you add a comment to remove any doubt that the intended char here is `}`?